sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/PluginReferentialTest.java

```diff
@@ -102,6 +103,22 @@ public void should_add_dependency() {
     assertThat(pluginReferential.findPlugin("bar").getRelease(Version.create("1.0")).getOutgoingDependencies().iterator().next().getVersion().getName()).isEqualTo("1.0");
   }
 
+
+  @Test
+  public void ignore_add_license_dependency() {
+    Plugin licensePlugin = Plugin.factory("license");
+    Release licenseRelease = new Release(licensePlugin, "1.0");
+    licensePlugin.addRelease(licenseRelease);
+
+    Plugin bar = Plugin.factory("bar");
+    Release bar10 = new Release(bar, "1.0");
+    bar.addRelease(bar10);
+
+    PluginReferential pluginReferential = PluginReferential.create(asList(bar, licensePlugin));
+    pluginReferential.addOutgoingDependency(bar10, "license", "1.0");
+    assertTrue(pluginReferential.findPlugin("bar").getRelease(Version.create("1.0")).getOutgoingDependencies().isEmpty());
```

Comment:
nitpick: the `Version` can be created outside the assertion and you can use `version.getName()`in the line above.

Also, an assertj style assertion is a bit easier to read:
```
assertThat(pluginReferential.findPlugin("bar").getRelease(version).getOutgoingDependencies()).isEmpty();
````